plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineStandardFacet.java

```diff
@@ -184,32 +184,24 @@ public static void installAllAppEngineRuntimes(IFacetedProject project, IProgres
     }
   }
 
-  public static org.eclipse.wst.server.core.IRuntime createAppEngineServerRuntime(IProgressMonitor monitor)
-      throws CoreException {
+  public static org.eclipse.wst.server.core.IRuntime createAppEngineServerRuntime(
+      IProgressMonitor monitor) throws CoreException {
     IRuntimeType appEngineRuntimeType =
         ServerCore.findRuntimeType(AppEngineStandardFacet.DEFAULT_RUNTIME_ID);
     if (appEngineRuntimeType == null) {
-      throw new NullPointerException("Could not find " + AppEngineStandardFacet.DEFAULT_RUNTIME_NAME + " runtime type");
+      throw new NullPointerException(
+          "Could not find " + AppEngineStandardFacet.DEFAULT_RUNTIME_NAME + " runtime type");
     }
 
     IRuntimeWorkingCopy appEngineRuntimeWorkingCopy
```

Comment:
Let's see... hope I understood your question correctly. I _think_ currently the runtime doesn't (and doesn't need to) get, set, or provide the Cloud SDK. [The `LocalAppEngineServerBehavior` directly creates an `CloudSdkAppEngineDevServer` instance.](https://github.com/GoogleCloudPlatform/google-cloud-eclipse/blob/master/plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerBehaviour.java#L338)